code/src/index.css

```diff
@@ -1,13 +1,280 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Leckerli+One&display=swap');
+
+body{
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  min-height: 100vh;
+  /* Photo by Karolina Grabowska: https://www.pexels.com/photo/white-and-gray-snow-covered-ground-4709467/*/ 
+  background-image: url('./images/background.jpg');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  font-family: 'Poppins', sans-serif;
+  padding: 0;
+}
+
+/* General styling */
+.container{
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
+  background-color: rgba(255, 255, 255, 0.626);
+  border-radius: 10px;
+  padding: 10%;
+}
+
```

Comment:
I love the styling Tika, you did an amazing job! really REALLY impressed! and i see you opted for flex this time, nice to switch it up (you and Neah will have to teach me more about GRID someday)